Assignment-1-6/index.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+include __DIR__ . '/form.php';
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> PHP Basic Assignment </title>
+  <link rel="stylesheet" href="./css/index.css">
+</head>
+
+<body>
+  <section class="hero">
+    <div class="login-main">
+      <!-- User's input form -->
+      <div class="user-input">
+        <form action="<?= $_FORM['php_self'] ?>" method="POST" enctype="multipart/form-data">
```

Comment:
Please just use this file for displaying the initial form.
For the output of the form, redirect to a differet page.